configs/vue3.js

```diff
@@ -11,11 +11,22 @@ module.exports = {
   },
   plugins: ['@typescript-eslint', 'vuejs-accessibility'],
   rules: {
+    'vue/no-empty-component-block': 'error',
     'vue/component-tags-order': [
       'error',
       {
         order: ['script', 'template', 'style'],
       },
     ],
+    'vue/block-lang': [
+      'error',
+      {
+        script: {
+          lang: 'ts',
+        },
+      },
+    ],
+    'vue/define-props-declaration': ['error', 'type-based'],
+    'vue/define-emits-declaration': ['error', 'type-based'],
```

Comment:
Is hier de aanname/vereiste dat alle vue3 projecten ook typescript projecten zijn?

Reply:
Ja, goede aanname lijkt mij. Maar snap dat dit misschien een harde beslissing is. We kunnen dit eventueel voor nu pauzeren en in een overleg beslissen.